uint16 serviceID
uint16 MethodID
uint32 dataLength
uint16 clientID
uint16 sessionID
uint8 protocolVersion
uint8 interfaceVersion
uint8 messageType
uint8 returnCode
uint64 crc
uint32 length
uint32 sqc
uint32 dataID
uint32 timestampNanoseconds
uint32 timestampSeconds
uint8 timestampSyncStatus
uint32 eventDataQualifier
uint8 extendedQualifier
uint16 originInvalidFlags
float32 originXPos
float32 originXSTD
float32 originYPos
float32 originYSTD
float32 originZPos
float32 originZSTD
float32 originRoll
float32 originRollSTD
float32 originPitch
float32 originPitchSTD
float32 originYaw
float32 originYawSTD
uint8 listInvalidFlags
detections[] detectionArray
float32 listRadVelDomainMin
float32 listRadVelDomainMax
uint32 listNumOfDetections
float32 alnAzimuthCorrection
float32 alnElevationCorrection
uint8 alnStatus